kratos/utilities/normal_calculation_utils.cpp

```diff
@@ -476,8 +481,10 @@ void NormalCalculationUtils::CalculateNormalsUsingGenericAlgorithm(
     auto& r_entities_array = GetContainer<TContainerType>(rModelPart);
     const auto it_entity_begin = r_entities_array.begin();
 
+    auto retrieve_normal = ConsiderUnitNormal ? [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) -> array_1d<double, 3> {return rGeometry.UnitNormal(rLocalCoordinates);} : [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) -> array_1d<double, 3> {return Coefficient * rGeometry.Normal(rLocalCoordinates);};
```

Comment:
```suggestion
    auto retrieve_normal = ConsiderUnitNormal 
        ? [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) 
            {return rGeometry.UnitNormal(rLocalCoordinates);} 
        : [](const GeometryType& rGeometry, const GeometryType::CoordinatesArrayType& rLocalCoordinates, const double Coefficient) 
            {return Coefficient * rGeometry.Normal(rLocalCoordinates);};
```

more readable formatting